src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ResidentJudgeFtpaDecisionConfirmationTest.java

```diff
@@ -237,6 +239,39 @@ void should_return_reheardRule35_confirmation() {
             .contains("#### What happens next");
     }
 
+    @Test
```

Comment:
This test should be a part of test method above "should_return_reheardRule35_confirmation"
E.g.

 @ParameterizedTest
    @MethodSource("toggleImaSwitchMode")
    void should_return_template_name(boolean toggleImaSwitchMode) {
        when(imaFeatureTogglerHandler.isImaEnabled()).thenReturn(toggleImaSwitchMode);

        if (toggleImaSwitchMode) {
            assertEquals(templateName, bailSubmissionTemplate.getName());
        } else {
            assertEquals(templateNameWithoutUt, bailSubmissionTemplate.getName());
        }
    }

    private static Stream<Arguments> toggleImaSwitchMode() {
        return Stream.of(
            Arguments.of(true),
            Arguments.of(false)
        );
    }